Sources/Fluent/Pivot/PivotProtocol.swift

```diff
@@ -59,8 +59,11 @@ extension PivotProtocol where Self: Entity {
         let rightId = try right.assertExists()
 
         try makeQuery()
-            .filter(Left.foreignIdKey, leftId)
-            .filter(Right.foreignIdKey, rightId)
+            .filter(pivotLeftIdKey, leftId)
+            .filter(pivotRightIdKey, rightId)
             .delete()
     }
 }
+
+public var pivotLeftIdKey: String = "left_id"
```

Comment:
We should use the foreign key name of the type (e.g., `user_id`) by default, not a generic `left_id`.